packages/server/docker-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+# TODO: wait for db server
+sleep 30
+
+npx sequelize db:create
```

Comment:
This guy seems to prevent container restarts, because it throws an error if database already exists